documentation/modules/deploying/proc-deploy-kafka-connect-using-kafka-connect-build.adoc

```diff
@@ -64,6 +64,13 @@ $ kubectl apply -f <kafka_connect_configuration_file>
 
 . Use the Kafka Connect REST API or `KafkaConnector` custom resources to use the connector plugins you added.
 
+.Rebuilding the container image with new artifacts
+
+To update a base image and download the latest connector plugin artifacts, you can trigger a rebuild of the container image associated with the Kafka Connect cluster by applying the annotation `strimzi.io/force-rebuild=true` to the Kafka Connect `StrimziPodSet` resource.
```

Comment:
Maybe we can do it like this to make it more clear you don't need this when you change the `KafkaConnect` CR?

```suggestion
A new container image is build automatically when you change the base image (`.spec.image`) or change the configured connector plugin artifacts configuration (`.spec.build.plugins`).

To pull an upgraded base image or to download the latest connector plugin artifacts without changing the `KafkaConnect` resource, you can trigger a rebuild of the container image associated with the Kafka Connect cluster by applying the annotation `strimzi.io/force-rebuild=true` to the Kafka Connect `StrimziPodSet` resource.
```